### ISSUE TITLE
[BUG] dds pkg init-db --help: uncaught exception instead of help message

### ISSUE THREAD

**Issue by Quincunx271** [vector-of-bool/dds#72]
**Describe the bug**

`dds pkg init-db --help` crashes with an exception. It should print a help message.

**Steps to Reproduce**

```
dds pkg init-db --help
```

**`dds` Output**

```
terminate called after throwing an instance of 'boost::leaf::leaf_detail::exception<debate::invalid_arguments>'
  what():  Invalid argument value given for enum-bound argument
Aborted (core dumped)
```

**Desktop (please complete the following information):**

 - `dds` Version: `0.1.0-alpha.6`, `develop`

**Comment by Dox5** [comment]
I've taken a look into this issue and I think I've identified what is wrong - I'm not sure exactly how to fix the problem though!
First it seems to happen for both `dds pkg init-db` and `dds pkg init-db --help` (or really anything past the init-db!)

Stepping through with GDB the exception starts here: https://github.com/vector-of-bool/dds/blob/develop/src/debate/enum.hpp#L34 because an invalid magic_enum conversion happens as there is a missing `init_db` from the `pkg_subcommands` enum here: https://github.com/vector-of-bool/dds/blob/develop/src/dds/cli/options.hpp#L35
Adding that enum value then shows that there is a missing case from dispatch_main: https://github.com/vector-of-bool/dds/blob/develop/src/dds/cli/dispatch_main.cpp#L43 and this is where I got stuck, it seems like there should be a `cmd::pkg_init_db` but I couldn't see anything along those lines and I don't know where to start! (I had a peek through the git history to see if there was a previous implementation that wasn't moved but I couldn't see anything obviously related).
I added a case that just throws and running `dds pkg init-db` or `dds pkg init-db --help` now get past the original point to my exception.

Happy to implement the command if I can get some pointers on where to start on the init DB side of things!

**Comment by Dox5** [comment]
It looks like this might now be fixed on develop (in 4a932ae708a6cf28caf961d301b861691b34821b) , I now get a nice error message:
```
./_build/dds pkg init-db --help
Usage: ./_build/dds pkg {ls,create,search,prefetch,solve}
Unrecognized argument/subcommand: "init-db"
  (Did you mean '-l'?)
```

**Comment by vector-of-bool** [comment]
Many things have changed about the CLI, so it's likely that this was incidentally fixed as part of the larger changes. I'll close this for now, but if a similar error appears in another location, we can refer back to this.
